spec/unit/provider/oneview_logical_downlink_synergy_spec.rb

```diff
@@ -84,7 +84,7 @@
       expect { provider.get_without_ethernet }.to raise_error(/The method #get_without_ethernet is unavailable for this resource/)
     end
 
-    xit 'should raise error when running get without ethernet - Test disabled due to bug on oneview-sdk' do
+    it 'should raise error when running get without ethernet - Test disabled due to bug on oneview-sdk' do
```

Comment:
Since the test was enabled, the `Test disabled due to bug on oneview-sdk` message doesn't make sense anymore